modules/end-user-guide/partials/proc_building-an-image-for-intellij-idea-community-or-pycharm-community.adoc

```diff
@@ -0,0 +1,99 @@
+[id="building-an-image-for-intellij-idea-community-or-pycharm-community_{context}"]
+= Building an image for IntelliJ IDEA Community or PyCharm Community
+
+This procedure describes how to build an image for IntelliJ IDEA Community or PyCharm Community.
+
+.Prerequisites
+
+* The build host has at least 2 GB of available RAM.
+
+* The build host has an installation of following tools:
```

Comment:
`The build host has an installation of following tools:` = `The build host makes an installation of following tools:`

Reply:
```suggestion
* The the following tools are installed on the build host:
```

Reply:
https://redhat-documentation.github.io/supplementary-style-guide/#general-formatting
_"When writing prerequisites, be as clear and concise as possible. **You can use the passive voice, if necessary, to achieve that end.**"_
BTW, "_Git is available_" is also passive voice.